testsuite/extra/pom.xml

```diff
@@ -53,8 +53,8 @@
       <artifactId>smallrye-metrics-testsuite-common</artifactId>
     </dependency>
     <dependency>
-      <groupId>javax.enterprise</groupId>
-      <artifactId>cdi-api</artifactId>
+      <groupId>jakarta.enterprise</groupId>
+      <artifactId>jakarta.enterprise.cdi-api</artifactId>
```

Comment:
Should this and the `api-tck` pom have the dependency set to provided?

Reply:
Maybe. It would feel more consistent from the POV that Jakarta artifacts in other modules are `provided`, but on the other hand it would feel *inconsistent* because all other dependencies in `testsuite/extra/pom.xml` and `testsuite/api-tck/pom.xml` use the default scope (they don't specify it).  Technically I don't think it matters at all for a testing-only module that we don't release.

Reply:
I asked because `testsuite/common/pom.xml` has the provided scope

Reply:
Hm that's a good observation. Ok, again I don't think it really matters but Iet's add `provided` here too then